roles/ckan/templates/kubernetes/ckan_cronjob.yaml

```diff
@@ -19,6 +19,7 @@ spec:
 
           restartPolicy: Never
 
+{% if enable_google_analytics is defined and enable_google_analytics %}
```

Comment:
I think we forgot to update this line with the new variable name?  We were also toying with removing the `is defined` part as well?

Reply:
Yes! Not sure how that slipped, and I look over the code again. Maybe I'm getting tired. Will fix it right away.